src/missing_tokens.py

```diff
@@ -60,25 +63,47 @@ def get_all_tokens(self) -> set[Address]:
         return set(self.v1 + self.v2)
```

Comment:
`is_empty` wasn't returning the correct value when the lists were empty on Gnosis network. Here's a suggestion for an update:

```suggestion
    def is_empty(self) -> bool:
        if not self.v1 and not self.v2:
            return True
```

Reply:
I can't comment on not edited lines unfortunately.

Reply:
Cool will fix. I guess we need to add a unit test here!

Reply:
What about just 

```py
return no self.v1 and not self.v2
```

Or ... I should be able to factor out the nots..

```py
return not (self.v1 or self.v2)
```

Reply:
I like `return no self.v1 and not self.v2`